modules/rosa-create-objects.adoc

```diff
@@ -861,6 +861,73 @@ Add a machine pool with tags to a cluster.
 $ rosa create machinepool --cluster=mycluster --replicas=2 --instance-type=r5.2xlarge --tags='foo bar,bar baz' --name=mp-1
 ----
 
+[id="rosa-create-network_{context}"]
+== create network
+
+Create a network that creates any necessary AWS resources through AWS CloudFormation templates. This helper command is intended to help create and configure a VPC for use with ROSA with HCP.
+
+[NOTE]
+====
+For custom or advanced configuration, it is highly recommended to use the AWS CLI directly using the `aws cloudformation` command.
+====
+
+[IMPORTANT]
+====
+Running this command creates resources within your AWS account.
+====
+
+.Syntax
+[source,terminal]
+----
+$ rosa create network [flags]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|<template-name>
+|Allows you to use a custom template. Templates must be in the template folder, structured as _templates/<template-name>/cloudformation.yaml_. If no template name is provided, command uses the default template.
```

Comment:
Make sure to include that for binary builds this template directory will have to be referenced manually after it is downloaded

Reply:
Added.